package.json

```diff
@@ -298,4 +298,4 @@
     "ansi-regex": "5.0.1",
     "trim-newlines": "5.0.0"
   }
-}
+}
```

Comment:
[pebble] Somehow the extra space at the end of the file was removed. My fault with my script most likely. Do you mind re-adding the blank line at the end of the file for me please?